packages/addons/addon-depends/chrome-depends/gtk3/package.mk

```diff
@@ -9,7 +9,9 @@ PKG_LICENSE="LGPL"
 PKG_SITE="http://www.gtk.org/"
 PKG_URL="https://ftp.gnome.org/pub/gnome/sources/gtk+/${PKG_VERSION:0:4}/gtk+-$PKG_VERSION.tar.xz"
 PKG_DEPENDS_TARGET="toolchain at-spi2-atk atk cairo gdk-pixbuf glib libX11 libXi libXrandr libepoxy pango"
+PKG_DEPENDS_CONFIG="libXft pango gdk-pixbuf shared-mime-info"
```

Comment:
Add `libXft` and `shared-mime-info` to `PKG_DEPENDS_TARGET`?

Reply:
Same as mediainfo, those are 2nd level dependencies. Some mechanism to do all that automatically would be nice, but not exactly trivial. Do we really wanna add all these deps?

Reply:
No that's fine, in that case we just have to hope that the packages that currently have `libXft` and `shared-mime-info` as dependencies don't drop those dependencies in future, otherwise this package will break. :smile: